tests/DynamoDbNonCompositeModelTest.php

```diff
@@ -1156,6 +1156,16 @@ public function testQueryNestedAttributes()
         $this->assertEquals($item['id']['S'], $results->first()->id);
     }
 
+    public function testBuilderContainsAllWhereClausesWhenGivenArrayOfConditions()
+    {
+        $builder = $this->getTestModel()->where([
+            "foo" => "bar",
+            "bin" => "baz"
+        ]);
+
+        $this->assertEquals(2, count($builder->wheres));
```

Comment:
Could you make it more explicit by comparing the full array instead of comparing the count only?

Reply:
Of course! Updated.